src/main/java/Duke.java

```diff
@@ -1,10 +1,251 @@
+        System.out.println("    ┌────────────────────────────────────────────────────────────────────┐");
+    }
+
+    /**
+     * Prints the bottom horizontal line to demarcate text from Tired.
+     */
+    public static void printBottomLine() {
+        System.out.println("    └────────────────────────────────────────────────────────────────────┘");
+    }
+
+    /**
+     * Prints logo of Tired
+     */
+    public static void printLogo() {
+        // Generated with: https://patorjk.com/software/taag/
+        String logo = "         ______       __     __                 __\n"
+                + "        / ____/___   / /_   / /   ____   _____ / /_\n"
+                + "       / / __ / _ \\ / __/  / /   / __ \\ / ___// __/\n"
+                + "      / /_/ //  __// /_   / /___/ /_/ /(__  )/ /_  \n"
+                + "      \\____/ \\___/ \\__/  /_____/\\____//____/ \\__/\n"
+                + "            ┌─┐┬  ┌─┐┌─┐┌─┐┌─┐┌─┐\n"
+                + "            ├─┘│  ├┤ ├─┤└─┐├┤  ┌┘\n"
+                + "      o o o ┴  ┴─┘└─┘┴ ┴└─┘└─┘ o";
+        System.out.println(logo);
+    }
+
+    /**
+     * Prints the list of tasks collated by Tired.
+     */
+    public static void printList() {
+        printTopLine();
+        System.out.println(INDENT + "Here are your tasks, \"oRgAnIc iTeLlIgEnCe\":");
+        for (int i = 0; i < taskCount; i++) {
+            System.out.println(INDENT + (i + 1) + "." + list[i]);
+        }
+        printBottomLine();
+    }
+
+    /**
+     *  Prints error message to user. Prompts user to input correct command.
+     */
+    public static void printError() {
+        printTopLine();
+        System.out.println(INDENT + "You didn't input the type of task. Again. Or you're stupid. Whatever.");
+        printBottomLine();
+    }
+
+    /**
+     * Prints error message for when user leaves out important information in input.
+     */
+    public static void printMissingTextError() {
+        printTopLine();
+        System.out.println(INDENT + "And?? Retype and complete your sentence like a grown adult. Please.");
+        printBottomLine();
+    }
+    /**
+     * Adds inputs from user to list[] to keep track of user's tasks, deadlines, and events.
+     *
+     * @param taskName Name of task from user.
+     * @param taskType Type of task from user.
+     * @param taskDetails Time/date of event/deadline.
+     */
+    public static void addTask(String taskName, String taskType, String taskDetails) {
+        switch (taskType) {
+        case "todo":
+            list[taskCount] = new Todo(taskName);
+            break;
+        case "deadline":
+            list[taskCount] = new Deadline(taskName, taskDetails);
+            break;
+        case "event":
+            list[taskCount] = new Event(taskName, taskDetails);
+            break;
+        default:
+            return;
+        }
+
+        taskCount++;
+        String plural = (taskCount - taskCompleted) == 1 ? "" : "s";
```

Comment:
Perhaps the name [could be self-explanatory](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#use-name-to-explain). Also, with `(taskCount - taskCompleted)` is used more than once, you can consider assigning to variables first? This would help [avoid complciated expressions](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-complicated-expressions).